### PR TITLE
no `1/ff(m0)`for the numerotor

### LINKED ISSUE #31: Undocumented
normalization is done in `TBDs.jl`. It's not suppose to be done
```
FF_Rk_svf = FF_Rk(p) * (1 / FF_Rk(p(mR^2)))
FF_ij_svf = FF_ij(q) * (1 / FF_ij(q(mR^2)))
```

non of the objects know about normalization. It should be in the coupling then

### PR DESCRIPTION
**Pull request by mmikhasenko**
Numerical factor is absorbed into the weight.

Closes #31

### CODE REVIEW

File: docs/julia/Manifest.toml
Comment (redeboer) [inline docs/julia/Manifest.toml:1]: Why are the dependencies updated?

File: models/Lb2pKg.json [hunk 1]
```diff
@@ -1,33 +1,33 @@
 {
   "distributions": [
     {
-      "name": "Lb2pKg_model",
+      "name": "default_model",
```
Comment (redeboer) [inline models/Lb2pKg.json:4]: I assume the intend is to implement the alternative models for this model in the same file?

**Review by redeboer — APPROVED** [review]
Checksums are okay, so approved :1st_place_medal: Some other reordering changes that seem unrelated to this issue.

I hope simply absorbing ff(0) is having no side effects to other places where the form factor is used. Energy-dependent with in particular, that can't be absorbed into weights.

File: models/Lb2pKg.json (hunk 1 above)
Comment (mmikhasenko) [reply models/Lb2pKg.json:4]: yes!

**Pull request merged**